pkg/products/rhsso/reconciler.go

```diff
@@ -195,6 +202,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	if err := r.reconcileConsoleLink(ctx, serverClient); err != nil {
```

Comment:
I think we should only be doing this on managed-api install type.

Reply:
Good point. Should we also be keeping the previous text for 3scale ("3scale API Management") when it's not managed-api?

Reply:
I think calling it API Management in RHMI 2.x is fine.

Reply:
I don't think it's used much in RHMI 2.x anyway, I'd say it's mostly accessed from the solution explorer.

Reply:
Cool, I just updated the PR with the condition to ensure the ConsoleLink is not created if the type isn't managed-api:
https://github.com/integr8ly/integreatly-operator/pull/1190/files#diff-74bcf7e72cc996549e6ed196e756ab37R410-R414